source/lac/petsc_precondition.cc

```diff
@@ -488,32 +490,62 @@ namespace PETScWrappers
     AssertThrow (ierr == 0, ExcPETScError(ierr));
 
     if (additional_data.output_details)
-      PetscOptionsSetValue("-pc_hypre_boomeramg_print_statistics","1");
+#if PETSC_VERSION_GT(3,7,0)
```

Comment:
I think implementing `DEAL_II_PETSC_VERSION_XXX` in terms of `PETSC_VERSION_XXX` makes good sense to me if possible.